Sommersemester/create_db.sql

```diff
@@ -48,8 +48,10 @@ CREATE TABLE Exercise (
     MuscleGroup NVARCHAR(50) NOT NULL,
     Sets INT NOT NULL,
     Repetitions INT NOT NULL,
+    OverExerciseId INT,
     Description NVARCHAR(100),
     PRIMARY KEY (Id)
+    FOREIGN KEY (OverExerciseId) REFERENCES Exercise(Id)
```

Comment:
SQL Error [156] [S0001]: Incorrect syntax near the keyword 'FOREIGN'.
  Incorrect syntax near the keyword 'FOREIGN'.
  Incorrect syntax near the keyword 'FOREIGN'.

Hint: missing , after primary key constraint